src/lib/airtable/request.js

```diff
@@ -89,11 +91,16 @@ export const createCustomer = async (customer) => {
       },
       body: JSON.stringify(customer)
     })
-
     console.log(resp);
+    await resp.json().then(data => customerId = data.id);
   } catch (err) {
     console.log(err);
+    customerId = generateOfflineId();
   }
+  customer.id = customerId;
+  addCustomerToRedux(customer);
+  setCurrentCustomerInRedux(customer);
```

Comment:
Since this isn't directly related to creating the customer, I think it might make more sense to move this to the actual `handleSubmit` right before navigating?

Reply:
Good catch, I agree. Thanks!